src/network.c

```diff
@@ -3477,7 +3477,7 @@ static int network_init (void)
 		status = plugin_thread_create (&dispatch_thread_id,
 				NULL /* no attributes */,
 				dispatch_thread,
-				NULL /* no argument */);
+				NULL /* no argument */, "network dispatch");
```

Comment:
This is 1 byte too long and will be truncated. Not a big issue, but perhaps we can come up with something shorter?